test/manual/amp-carousel.amp.html

```diff
@@ -89,5 +89,11 @@
     <amp-img src="https://lh3.googleusercontent.com/5rcQ32ml8E5ONp9f9-Rf78IofLb9QjS5_0mqsY1zEFc=w400-h300-no" width=400 height=300></amp-img>
     <amp-img src="https://lh3.googleusercontent.com/Z4gtm5Bkxyv21Z2PtbTf95Clb9AE4VTR6olbBKYrenM=w400-h300-no" width=400 height=300></amp-img>
   </amp-carousel>
+
+  <amp-carousel id="carousel-4" width=400 height=300 type=slides controls indicators>
```

Comment:
@dvoytenko sorry, cant seem to remember having that discussion. we removed the "dots" functionality a while ago right w/c is similar to the to "indicators"?